### PR TITLE
using stepic API via python3

### PR DESCRIPTION
**Pull request by eanioner**
jetbrains internship

### CODE REVIEW

**Comment by eanioner** [comment]
Python and stepicAPI

File: examples/user_courses.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                         data={'grant_type': 'client_credentials'},
+                         auth=auth
+                         )
+            self.token = json.loads(resp.text)['access_token']
+        except:
+            print("Error while obtaining token")
+      
+    def get(self, url):
+        try:
+            resp = json.loads(requests.get(url, headers={'Authorization': 'Bearer ' + self.token}).text)
```
Comment (psviderski) [inline examples/user_courses.py]: A `Response` object has a special `json()` method that performs this `json.loads(...text)` trick.

File: examples/user_courses.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        except:
+            print("Error while getting data")
+            resp = None
+        return resp
+     
+    # announcements
+    def announcements(self, page = 1):
+        url = self.api_url + 'announcements?page=%d'%page
```
Comment (psviderski) [inline examples/user_courses.py]: It's preferable to use a new format string syntax: https://docs.python.org/3.4/library/string.html#format-examples

File: examples/user_courses.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        except:
+            print("Error while getting data")
+            resp = None
+        return resp
+     
+    # announcements
+    def announcements(self, page = 1):
+        url = self.api_url + 'announcements?page=%d'%page 
+        return self.get(url)
+    def announcements_pk(self, pk):
```
Comment (psviderski) [inline examples/user_courses.py]: Insert a blank line between method definitions: https://www.python.org/dev/peps/pep-0008/#blank-lines

File: examples/user_courses.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    def course_subscriptions(self, page = 1):
+        url = self.api_url + 'course-subscriptions?page=%d'%page
+        return self.get(url)
+    def course(self, course_id):
+        url = self.api_url + 'courses/%d'%course_id
+        return self.get(url)
+    
+        
+sapi = StepicAPI(client_id, client_secret)
+#Пример получения списка курсов
```
Comment (psviderski) [inline examples/user_courses.py]: English comments are more preferable.

File: examples/user_courses.py (hunk 1 above)
Comment (eanioner) [inline examples/user_courses.py]: fixed

File: examples/user_courses.py (hunk 2 above)
Comment (eanioner) [inline examples/user_courses.py]: fixed

File: examples/user_courses.py (hunk 3 above)
Comment (eanioner) [inline examples/user_courses.py]: fixed

File: examples/user_courses.py (hunk 4 above)
Comment (eanioner) [inline examples/user_courses.py]: fixed

**Comment by mehanig** [comment]
Looks good, merged

**Comment by taery** [comment]
@eanioner, можете еще сделать небольшую html страницу, отображающую результат вашего пул-реквеста?

**Pull request merged**